punc/include/punc/population.h

```diff
@@ -403,7 +403,7 @@ class Population
     std::size_t num_of_particles();         ///< Returns number of particles
     std::size_t num_of_positives();         ///< Returns number of positively charged particles
     std::size_t num_of_negatives();         ///< Returns number of negatively charged particles
-
+    void statistics(double *stats);         ///< Returns mean speed and standard deviation for each species
```

Comment:
Since this returns values using a parameter array it needs a slightly bigger docstring explaining what the output is (i.e. what the different elements in `stats` is)

Reply:
Alright, I'll extend the docstring.